wp-downloadmanager.php

```diff
@@ -1509,7 +1509,7 @@ function downloadmanager_activate() {
 							"file_updated_date varchar(20) NOT NULL default '',".
 							"file_last_downloaded_date varchar(20) NOT NULL default '',".
 							"file_hits int(10) NOT NULL default '0',".
-							"file_permission TINYINT(2) NOT NULL default '0',".
+							"file_permission varchar(255) NOT NULL default '0',".
```

Comment:
If we change the col, we need to provide exisiting user an upgrade path.

Similar to https://github.com/lesterchan/wp-polls/blob/master/wp-polls.php#L1881-L1888